src/main/java/org/broadinstitute/hellbender/utils/bwa/Alignment.java

```diff
@@ -0,0 +1,142 @@
+package org.broadinstitute.hellbender.utils.bwa;
+
+import htsjdk.samtools.*;
+import htsjdk.samtools.util.QualityUtil;
+import htsjdk.samtools.util.SequenceUtil;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.BaseUtils;
+import org.broadinstitute.hellbender.utils.QualityUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.SAMRecordToGATKReadAdapter;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Alignment {
```

Comment:
I'm going to declare that creating an appropriate SAMFileHeader for the BAM that's written by BwaSpark after doing the aligning is out of scope for this PR.  It's that program that's broken, it's not broken any worse than it was, and the breakage doesn't really have anything to do with alignment.  Reviewers OK with that?